README.md

```diff
@@ -159,6 +159,26 @@ node.walk([options], action, [context])
 
 These functions can also take, as the last parameter, the *context* on which the action will be called.
 
+## Typescript Refrence
+
+You can maually find the definition files in the `types` folder. A quick summary, is rougly as follows:
+
+```ts
+import * as TreeModel from "tree-model";
+
+interface TestModel {
+    name: string;
```

Comment:
I would like a more complete example that also type checks the children, maybe:

```suggestion
    name: string,
    children?: Array<TestModel>
```

But I don't know enough TypeScript...